plugins/CoreHome/vue/src/index.ts

```diff
@@ -92,7 +92,7 @@ export { default as SideNav } from './SideNav/SideNav';
 export { default as EnrichedHeadline } from './EnrichedHeadline/EnrichedHeadline.vue';
 export { default as ContentBlock } from './ContentBlock/ContentBlock.vue';
 export { default as Comparisons } from './Comparisons/Comparisons.vue';
-export { default as MenuDropdown } from './MenuDropdown/MenuDropdown.vue';
```

Comment:
Technically this is a bit of a BC break if a plugin uses MenuDropdown. It could potentially also be exported as MenuDropdown and MenuItemsDropdown.

No plugin probably uses it, though, so it's probably fine.

Reply:
@diosmosis but a plugin might need to use the vue component, right? The Angular JS component should still work as before.
I've looked through all our plugins and it wasn't used. I doubt any third party plugin is already migrated to vue...

Reply:
Yes a plugin might need to use it which is why changing the export name (MenuDropdown -> MenuItemsDropdown) can be seen as a BC break. It's not important here but worth keeping in mind for the future I think after Matomo 5 when plugins may use them directly.